package.json

```diff
@@ -91,6 +91,8 @@
     "@types/react-dates": "21.8.1",
     "@types/react-dom": "17.0.0",
     "@types/react": "17.0.0",
+    "@types/react-is": "17.0.0",
+    "@types/resize-observer-browser": "0.1.5",
```

Comment:
How come we need those types? Pretty sure I remove all the polyfill code for resize-observe and only use the native api

Reply:
The timing is a bit awkward. ResizeObserver type was and is not yet typed in typescript (in the lib.dom.d.ts file).

It's been added in the TypeScript Repo 27 days ago, but there hasn't been a release yet. Most likely, in Typescript 4.1.4 or 4.2, the types will be present and we won't need to import this package. 

TypeScript DOM Lib generator feature link : https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/847